packages/styletron-engine-atomic/src/client/client.js

```diff
@@ -222,3 +228,18 @@ class StyletronClient implements StandardEngine {
 }
 
 export default StyletronClient;
+
+function findSheetIndexWithMedia(children, media) {
+  let index = 0;
+  for (; index < children.length; index++) {
+    const child = children[index];
+    if (
+      child.tagName === "STYLE" &&
+      ((child: any): HTMLStyleElement).media === media
+    ) {
+      return index;
+    }
+  }
+
+  return -1;
```

Comment:
`Array.prototype.findIndex` returns `-1` if not found, so I've preserved this here.

Should we handle the case that it doesn't exist for some reason? This should never happen, but if it did, we'd attempt array access at `-1` which would throw.

Reply:
This is still fine even with `-1`. `children[-1]` returns `undefined` and `insertBefore(element, undefined)` just appends the element to the end:

> If the reference node is null, the specified node is added to the end of the list of children of the specified parent node.

Reply:
Ah, TIL